Pipfile

```diff
@@ -35,6 +35,7 @@ pydocstyle = "*"
 pytest = "*"
 pytest-cov = "*"
 pytest-randomly = "*"
+pytest-lazy-fixture = "*"
```

Comment:
@zupo 
This allows you to use fixtures inside of `pytest.mark.parametrize` or `pytest.fixture(params=(...))`
See: [usage](https://github.com/Pylons/pyramid_openapi3/pull/150/files#diff-0b343715cce789e1605576bb6c7e10d237fd55fa21839807a2c674ce725900faR135) or [library](https://github.com/tvorog/pytest-lazy-fixture)

---

I see Circle CI failing due to it not being installed. I faintly remember you saying that something needs to be done to bust the CI cache

Reply:
Oh, actually it seems like it is the typecov / mypy failing. Will take a look

Reply:
Okay got it working. I needed to use
```python
from pytest_lazyfixture import lazy_fixture
```
over
```python
pytest.lazy_fixture
```
which is a known issue WRT typing and pytest plugins

---

Secondly, I just had to install the typeshed (typing stubs) for it (see [here](https://github.com/python/typeshed/tree/master/stubs/pytest-lazy-fixture))